src/test/java/posters/pageobjects/pages/browsing/ProductdetailPage.java

```diff
@@ -29,9 +29,10 @@
 
     @Override
     @Step("ensure this is a product detail page")
-    public void isExpectedPage()
+    public ProductdetailPage isExpectedPage()
     {
         $("#addToCartForm").should(exist);
```

Comment:
Please call `super.isExpectedPage();` at first in case we would add some default functionality.